composer.json

```diff
@@ -21,14 +21,14 @@
         "ext-curl":"*",
         "phpcr/phpcr": "~2.1.2",
         "phpcr/phpcr-utils": "^1.2.8",
-        "jackalope/jackalope": "~1.2.5"
+        "jackalope/jackalope": "~1.3"
```

Comment:
sorry, lets keep this at `~1.3.0` - otherwise things like we just did with adding methods to the transport interface won't be possible anymore.